docs/tables/aws_s3tables_namespace.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Steampipe Table: aws_s3tables_namespace - Query AWS S3 Tables Namespaces using SQL"
+description: "Allows users to query AWS S3 Tables Namespaces, providing information about the configuration, settings, and properties of your S3 namespaces."
+folder: "S3Tables"
+---
+
+# aws_s3tables_namespace
```

Comment:
Please follow the format mentioned in https://github.com/turbot/steampipe-plugin-aws/blob/main/docs/tables/aws_amplify_app.md#table-aws_amplify_app---query-aws-amplify-apps-using-sql